config/logback.xml

```diff
@@ -2,7 +2,9 @@
   <conversionRule conversionWord="highlightdark" converterClass="org.contikios.cooja.util.LogbackColor" />
   <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
     <encoder>
-      <pattern>${logback.layoutPattern:-%highlightdark(%-5level [%thread] [%file:%line] - %msg%n)}</pattern>
+      <!-- Highlight needs to finish before %n, or gradle will bleed into
+           console. Without space between ) and %n, output is a single line. -->
+      <pattern>${logback.layoutPattern:-%highlightdark(%-5level [%thread] [%file:%line] - %msg) %n}</pattern>
```

Comment:
An alternative to the workaround with space is to use an empty option list:

`${logback.layoutPattern:-%highlightdark(%-5level [%thread] [%file:%line] - %msg){}%n}`

This would avoid the ending whitespace on all log lines.

Reply:
Much better, fixed!